metadata-db-app/src/cmr/metadata_db/migrations/080_add_generic_document_table.clj

```diff
@@ -12,7 +12,7 @@
     provider_id VARCHAR(10) NOT NULL,
     document_name VARCHAR(20) NOT NULL,
     schema VARCHAR(255) NOT NULL,
-    format VARCHAR(255) NOT NULL,
+    format VARCHAR(255),
```

Comment:
why are we going to let format be empty?

Reply:
On the delete right now anyway, I have no idea how to get format in there.  Is there a format for something that is deleted?  If you want for deleted concepts to have a format, please write a ticket.  This will be a couple of days worth of work.

Reply:
maybe lets write a ticket to restore this

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8615